pkg/kontainer-engine/drivers/gke/gke_driver.go

```diff
@@ -786,7 +781,12 @@ func (d *Driver) PostCheck(ctx context.Context, info *types.ClusterInfo) (*types
 		return nil, err
 	}
 
-	svc, err := d.getServiceClient(ctx, state)
+	ts, err := GetTokenSource(ctx, state.CredentialContent)
```

Comment:
this can be replaced with `getServiceClient(ctx, state.CredentialContent)` like in the update method?

Reply:
We need the token source later for the `ServiceAccountToken`.